### ISSUE TITLE
phaxio.fax.create(faxParams) on Android causes application to crash

### ISSUE THREAD

**Issue by jjohnson382** [phaxio/phaxio-java#25]
I'm using my testing key and secret to instantiate a Phaxio object. I'm then putting a url to a PDF document stored in Google Firebase Storage in my fax params, and then creating a fax. 
```
        final Phaxio phaxio = new Phaxio(key, secret);
        String url = urls.get(0);
        HashMap<String, Object> faxParams = new HashMap<>();
        faxParams.put("to", OUTGOING_FAX_NUM);
        faxParams.put("content_url", url);
        Fax fax = phaxio.fax.create(faxParams);

```

When I press the button that I've linked to this `sendFax()` function, the application crashes. When looking through the LogCat, I found the error:
```
    java.lang.NoClassDefFoundError: Failed resolution of: Ljavax/xml/bind/DatatypeConverter;
```

From what I can understand, Android doesn't support javax.xml.bind.DatatypeConverter. Any possible fix for this? Thanks.

**Comment by noelherrick** [comment]
The new version 0.4.2 is out on Maven (it should be available in a few hours).

**Comment by jjohnson382** [comment]
I looked at `phaxio/restclient/BasicAuthorization.java` and it seems as if the file has not been updated on master or on Maven.

**Comment by noelherrick** [comment]
@jjohnson382 The PR was mistakenly closed instead of merged. Sorry for the confusion! I've published 0.4.3 with those changes.
